src/cli/cli.js

```diff
@@ -124,25 +120,17 @@ async function runTests (argParser) {
 }
 
 async function listBrowsers (providerName) {
-    const provider = await browserProviderPool.getProvider(providerName);
-
-    if (!provider)
-        throw new GeneralError(RUNTIME_ERRORS.browserProviderNotFound, providerName);
-
-    if (provider.isMultiBrowser) {
-        const browserNames = await provider.getBrowserList();
-
-        await browserProviderPool.dispose();
-
-        if (providerName === 'locally-installed')
+    try {
+        const browserNames = await createTestCafe.listBrowsers(providerName);
+        if (providerName == 'locally-installed')
             console.log(browserNames.join('\n'));
         else
-            console.log(browserNames.map(browserName => `"${providerName}:${browserName}"`).join('\n'));
+            console.log(browserNames.map(n => providerName+':'+n).join('\n'));
```

Comment:
Could you please keep `browserName` as the parameter name in the arrow function?